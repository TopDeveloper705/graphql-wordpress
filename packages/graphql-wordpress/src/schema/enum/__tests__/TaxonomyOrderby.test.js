import TAXONOMY_ORDERBY from 'enum/TaxonomyOrderby';

describe('Test enum values', () => {
  test('TAXONOMY_ORDERBY values', () => {
    expect(TAXONOMY_ORDERBY.getValues()).toMatchSnapshot();
  });
});
