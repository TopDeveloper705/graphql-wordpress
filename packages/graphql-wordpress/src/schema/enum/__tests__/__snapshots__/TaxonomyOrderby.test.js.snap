// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test enum values TAXONOMY_ORDERBY values 1`] = `
Array [
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "ID",
    "value": "id",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "INCLUDE",
    "value": "include",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "NAME",
    "value": "name",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "SLUG",
    "value": "slug",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "TERM_GROUP",
    "value": "term_group",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "DESCRIPTION",
    "value": "description",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "COUNT",
    "value": "count",
  },
]
`;
